source/vstd/array.rs

```diff
@@ -9,7 +9,9 @@ verus! {
 impl<T, const N: usize> View for [T; N] {
     type V = Seq<T>;
```

Comment:
Should we mark the view of arrays to be just `[T; N]` (like those primitive integer types)?

Reply:
My guess is that a sequence view will be more useful, since the spec writer can then make use of all the features in the sequence library.